Platform/ADLINK/ComHpcAltPkg/Library/MmcLib/MmcLib.c

```diff
@@ -0,0 +1,541 @@
+/** @file
+  MMC Library.
+
+  Copyright (c) 2022, ADLink. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MmcLib.h>
+#include <Library/NVLib.h>
+#include <Library/PcdLib.h>
+#include <Library/PL011UartLib.h>
+#include <Library/PrintLib.h>
+#include <Library/UefiLib.h>
+
+EFI_STATUS
+MmcPostCode (
+  IN UINT32  Value
+  )
+{
+  UINTN  NumberOfBytes;
+  UINT8  IpmiCmdBuf[]   = { "[C0 00 80 11]\r\n" };
+  UINTN  IpmiCmdBufSize = sizeof (IpmiCmdBuf);
+
+  AsciiSPrint ((CHAR8 *)IpmiCmdBuf, sizeof (IpmiCmdBuf), "[C0 00 80 %2X]\r\n", (UINT8)Value);

```

Comment:
I wonder how we convert 32-bit status code to 8-bit

Reply:
I see what it's doing now. It sends the post code to the MMC to display on the two 8-segment displays on the board. Since it's two digits, it can only display 8 bits. So it uses the lower 8 bits of the code that's passed in.

Reply:
I guess ADLINK found them useful, so I think let's keep it as is?

Reply:
Ok